tests/test_skip_report.py

```diff
@@ -0,0 +1,23 @@
+import os
+import subprocess
+
+from utils.command import build_command
+
+
+def test_skip_report(analysis_data):
```

Comment:
Looks good , how about putting this test in test_advanced_options.py where we can add more tests for other options like --csvreport etc .

Reply:
Solved in 234030b39d2ce20b3d02acb78011ddb65ee2fe5a.